z00_lmod.sh

```diff
@@ -5,71 +5,72 @@
 #
 ########################################################################
 
-if [ -z "${USER_IS_ROOT:-}" ]; then
+# Skip Lmod initialization if USER_IS_ROOT or LMOD_DISABLE is set
 
-  if [ -z "${MODULEPATH_ROOT:-}" ]; then
-    export USER=${USER-${LOGNAME}}  # make sure $USER is set
-    export LMOD_sys=`uname`
+([ -n "$USER_IS_ROOT" ] || [ -n "$LMOD_DISABLE" ]) && exit
```

Comment:
watch out!

the shell semantics for `||` are not the same like in C or C++

I'd expect the above code to interpret this as "if last condition check failed try next one")

If I was you, I'd simply copy-paste the previous clauses (duplication, but safe)